### PR TITLE
Update murus.rb

### PR DESCRIPTION
**Pull request by daylightbrightledlight**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

I've updated the zaps and deletes

### CODE REVIEW

File: Casks/murus.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
   zap delete: [
                 '/Library/Application Support/Murus',
+                '/Library/LaunchDaemons/it.murus.murusfirewallrules.plist',
```
Comment (commitay) [inline Casks/murus.rb]: This should be `uninstall launchctl`

**Review by commitay — DISMISSED** [review]

**Comment by commitay** [comment]
https://github.com/caskroom/homebrew-cask/pull/38152#issuecomment-326192869

**Comment by daylightbrightledlight** [comment]
Should the preference files be in `zap trash` instead of `zap delete`?

**Comment by daylightbrightledlight** [comment]
@commitay Do I have to add the name and version of the cask in the commit message if this is not a version update? Should I add the cask name and version on the first line?

**Pull request merged**